src/component/MainSection/FolderNameSection/FolderNameSection.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import "./FolderNameSection.css";
+
+const FolderNameSection = () => {
+  const [folderData, setFolderData] = useState(null);
+
+  useEffect(() => {
+    fetch("https://bootcamp-api.codeit.kr/api/sample/folder")
+      .then((response) => response.json())
+      .then((data) => {
+        setFolderData(data);
+      })
```

Comment:
```suggestion
      .then((data) => {
        setFolderData(data.folderData);
      })
```

이렇게 해주면 아래에서 folderData 상태를 참조할 때 더 간결할 것 같아요 :)